modules/ztp-sno-siteconfig-config-reference.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+:_content-type: REFERENCE
+[id="ztp-sno-siteconfig-config-reference_{context}"]
+= {sno-caps} SiteConfig CR installation reference
+
+.SiteConfig CR installation options for {sno} clusters
+[cols="1,3", options="header"]
+|====
+|SiteConfig CR field
+|Description
+
+|`spec.cpuPartitioningMode`
+|Configure workload partitioning by setting the value for `cpuPartitioningMode` to `AllNodes`.
+To complete the configuration, specify the `isolated` and `reserved` CPUs in the `PerformanceProfile` CR.
+You enable workload partitioning during cluster installation only.
+
+|`metadata.name`
+|Set `name` to `assisted-deployment-pull-secret` and create the `assisted-deployment-pull-secret` CR in the same namespace as the `SiteConfig` CR.
+
+|`clusterImageSetNameRef`
+|Configure the image set available on the hub cluster.
+To see the list of supported versions on your hub cluster, run `oc get clusterimagesets`.
+
+|`installConfigOverrides`
+a|Set the `installConfigOverrides` field to enable or disable optional components prior to cluster installation.
+[NOTE]
+====
+`NodeTuning` is required for 4.13 and later.
+====
+
+|`spec.clusters.clusterLabels`
+|Configure cluster labels to correspond to the `bindingRules` field in the `PolicyGenTemplate` CRs that you define.
+For example, `policygentemplates/common-ranGen.yaml` applies to all clusters with `common: true` set, `policygentemplates/group-du-sno-ranGen.yaml` applies to all clusters with `group-du-sno: ""` set.
+
+|`spec.clusters.crTemplates.KlusterletAddonConfig`
+|Optional. Set `KlusterletAddonConfig` to `KlusterletAddonConfigOverride.yaml to override the default `KlusterletAddonConfig` that is created for the cluster.
+
+|`spec.clusters.nodes.hostName`
+|For single-node deployments, define a single host.
+For three-node deployments, define three hosts.
+For standard deployments, define three hosts with `role: master` and two or more hosts defined with `role: worker`.
+
+|`spec.clusters.nodes.bmcAddress`
+|BMC address that you use to access the host. Applies to all cluster types. {ztp} supports iPXE and virtual media booting by using Redfish or IPMI protocols. To use iPXE booting, you must use {rh-rhacm} 2.8 or later. For more information about BMC addressing, see the "Additional resources" section.
+
+|`spec.clusters.nodes.biosConfigRef`
+|Optional. Configure required host BIOS settings.
```

Comment:
I believe that this is not yet a GA feature. @tliu2021 can you confirm please.

Reply:
Yes, this is not yet a GA feature and it should be removed from the custom doc.